jwst/ami/lg_model.py

```diff
@@ -107,24 +110,25 @@ def __init__(
 
     def simulate(self, fov, psf_offset=(0, 0)):
         """
-        Simulate a detector-scale psf.
+        Simulate a detector-scale PSF.
 
         Use parameters input from the call and
-        already stored in the object, and generate a simulation fits header
-        storing all of the  parameters used to generate that psf.  If the input
-        bandpass is one number it will calculate a monochromatic psf.
+        already stored in the object, and generate a simulation FITS header
+        storing all of the  parameters used to generate that PSF.  If the input
+        bandpass is one number, it will calculate a monochromatic PSF.
 
         Parameters
         ----------
         fov : int
             Number of detector pixels on a side
-        psf_offset : detector pixels
-            Center offset from center of array
+        psf_offset : tuple of int
+            Detector pixels defining the center offset from center of array
```

Comment:
```suggestion
            Offset from center of array in units of detector pixels.
```

Reply:
Hmm GH won't let me apply this directly. I can do it manually later.